src/Charts/BentoPie.tsx

```diff
@@ -119,16 +117,30 @@ const BentoPie = ({
   );
 };
 
-const RenderLabel: PieProps['label'] = (params) => {
-  const { cx, cy, midAngle, outerRadius, fill, payload, index, activeIndex } = params;
+const toNumber = (val: number | string | undefined, defaultValue?: number): number => {
+  let value = defaultValue || 0;
```

Comment:
i dont think this needs to be mutable / exist as a variable. this could be simplified to something like

```js
if (val && typeof val === "string") {
  return Number(val);
}
return defaultValue || 0;
```